src/persistent_tasks.jl

```diff
@@ -29,10 +34,10 @@ function test_persistent_tasks(package::Module; kwargs...)
     test_persistent_tasks(PkgId(package); kwargs...)
 end
 
-function has_persistent_tasks(package::PkgId; tmax = 10)
+function has_persistent_tasks(package::PkgId; expr::Expr = quote nothing end, tmax = 10)
```

Comment:
Actually, I think this is even simpler:
```suggestion
function has_persistent_tasks(package::PkgId; expr::Expr = :(), tmax = 10)
```